GitUI/Editor/ContinuousScrollEventManager.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using GitCommands;
+
+namespace GitUI.Editor
+{
+    public class ContinuousScrollEventManager
+    {
+        public EventHandler BottomScrollReached;
+        public EventHandler TopScrollReached;
+        public DateTime LastScrollEventFiredDate { get; set; } = DateTime.MinValue;
+
+        private bool IsScrollTooFast()
+        {
+            return DateTime.Now - LastScrollEventFiredDate < TimeSpan.FromMilliseconds(AppSettings.AutoViewScrollDelay);
+        }
+
+        public void CallBottomScrollReachedEvent(object sender, EventArgs e)
```

Comment:
```suggestion
        public void RaiseBottomScrollReached()
```
This aligns with Windows Forms vocabulary

Reply:
ok